moveit_ros/planning_interface/planning_scene_interface/src/wrap_python_planning_scene_interface.cpp

```diff
@@ -110,6 +110,104 @@ class PlanningSceneInterfaceWrapper : protected py_bindings_tools::ROScppInitial
     py_bindings_tools::deserializeMsg(ps_str, ps_msg);
     return applyPlanningScene(ps_msg);
   }
+
+  bool applyCollisionObjectPython1(const std::string& co_str)
```

Comment:
Where are these new functions used? Shouldn't they be registered with the Python wrapper?

Reply:
Whoops! How embarrassing. I'll push a new set of commits tomorrow.